actions/run.sh

```diff
@@ -1,21 +1,8 @@
 #!/bin/bash
 
-#: Title : run
-#: Date : 27/feb/2017
-#: Author : Daniel Ortega @gatero <me@daniel-ortega.mx>
-#: Version : 0.0.3
-#: Description : Run the project using docker-compose
-#: Options : 
-#:    -f) use your own docker-file
-#:    -d) default docker-compose
-#:    -h|*) help option
-
-#: Type : function
-#: Description : execute any command inside the container
-run_in_api() {
-  printf "%b" "${GREEN_REGULAR}${NO_COLOR}${DOCKER_CONTAINER_API}"
-  docker exec "$DOCKER_CONTAINER_API" "$@"
-}
+#DOCKER_CONTAINERS=`docker container ls -a --format "table {{.ID}} {{.Names}}"`
+declare -a DOCKER_CONTAINERS=(`docker container ls --format "{{.Names}}"`)
+#echo $DOCKER_CONTAINERS
```

Comment:
The same above comment, we need prevent has commented code lines in the source, the only way to let commented lines is as a part of documentation